bears/vhdl/VHDLLintBear.py

```diff
@@ -48,4 +57,5 @@ def check_prerequisites(cls):  # pragma: no cover
 
     @staticmethod
     def create_arguments(filename, file, config_file):
-        return which('bakalint.pl'), '--input', filename
+        return ('tar.gz', self.vdhl_tar_file, which('bakalint.pl'),
```

Comment:
So this would try to run `perl tar.gz bakalint.tar.gz ...` which would fail. badly.